nativelib/src/main/scala/scala/scalanative/native/FunctionPtr.scala

```diff
@@ -0,0 +1,1279 @@
+// ###sourceLocation(file: "/Users/Denys/.src/native/nativelib/src/main/scala/scala/scalanative/native/FunctionPtr.scala.gyb", line: 1)
+package scala.scalanative
+package native
+
+import scalanative.runtime.undefined
+import scala.reflect.ClassTag
+
+/** C-style function pointer. */
+sealed abstract class FunctionPtr
+
+object FunctionPtr {
+
+// ###sourceLocation(file: "/Users/Denys/.src/native/nativelib/src/main/scala/scala/scalanative/native/FunctionPtr.scala.gyb", line: 14)
+
```

Comment:
That's an artifact of our current code generation tool that we borrowed from Swift (called gyb.) There is an issue to replace it with something better (https://github.com/scala-native/scala-native/issues/135) but that's out-of-scope for M3.